streams/src/test/java/org/apache/kafka/streams/state/internals/TimeOrderedCachingPersistentWindowStoreTest.java

```diff
@@ -144,45 +148,50 @@ public void closeStore() {
     @SuppressWarnings("deprecation")
     @Test
     public void shouldDelegateDeprecatedInit() {
-        final RocksDBTimeOrderedWindowStore inner = EasyMock.mock(RocksDBTimeOrderedWindowStore.class);
-        EasyMock.expect(inner.hasIndex()).andReturn(hasIndex);
-        EasyMock.replay(inner);
-        final TimeOrderedCachingWindowStore outer = new TimeOrderedCachingWindowStore(inner, WINDOW_SIZE, SEGMENT_INTERVAL);
+        final RocksDBTimeOrderedWindowStore inner = mock(RocksDBTimeOrderedWindowStore.class);
+        when(inner.hasIndex()).thenReturn(hasIndex);
+
+        final TimeOrderedCachingWindowStore outer =
+                new TimeOrderedCachingWindowStore(inner, WINDOW_SIZE,
+                        SEGMENT_INTERVAL);
 
-        EasyMock.reset(inner);
-        EasyMock.expect(inner.name()).andStubReturn("store");
+        reset(inner);
+        when(inner.name()).thenReturn("store");
         inner.init((org.apache.kafka.streams.processor.ProcessorContext) context, outer);
```

Comment:
You need to remove this call since this call is only needed by EasyMock to record the calls to verify. Mockito verifies calls with `verify(inner).init((org.apache.kafka.streams.processor.ProcessorContext) context, outer)`. In this and the next line you are basically calling a method on the mock directly and verifying that you called the method on the mock. That does actually not add anything to the test. In a test you should verify that a call to a method of the class under test calls the method on the mock, not that the test calls directly the method on the mock.
So you can remove both line. 
After that you need also to remove the `times(2)` in `verify(inner, times(2)).init((org.apache.kafka.streams.processor.ProcessorContext) context, outer);` since that is only needed, because you call the method on the mock directly.

Reply:
So this test should look like this:
```java
    @Test
    public void shouldDelegateDeprecatedInit() {
        final RocksDBTimeOrderedWindowStore inner = mock(RocksDBTimeOrderedWindowStore.class);
        when(inner.hasIndex()).thenReturn(hasIndex);

        final TimeOrderedCachingWindowStore outer = new TimeOrderedCachingWindowStore(inner, WINDOW_SIZE, SEGMENT_INTERVAL);
        outer.init((org.apache.kafka.streams.processor.ProcessorContext) context, outer);

        verify(inner).init((org.apache.kafka.streams.processor.ProcessorContext) context, outer);
    }
```